indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunnerFactory.java

```diff
@@ -53,7 +64,10 @@ public RemoteTaskRunnerFactory(
       final ObjectMapper jsonMapper,
       @Global final HttpClient httpClient,
       final Supplier<WorkerBehaviorConfig> workerConfigRef,
-      ScheduledExecutorFactory factory
+      final ScheduledExecutorFactory factory,
+      final SimpleResourceManagementConfig config,
+      final ResourceManagementSchedulerConfig resourceManagementSchedulerConfig,
+      final ScheduledExecutorService exec
```

Comment:
@drcrallen are you able to start overlord after this change, guice does not seem to be able to inject "ScheduledExecutorService" .